backend/app/routes/user.py

```diff
@@ -270,6 +271,9 @@ def put(self):
                 return {"error": User.error_email_already_exists}, 400
 
             user = self.register_user(email, app_version=app_version, password=password, name=name)
+            if admin_creation:
+                user.onboarding_presented = datetime.now()
+                user.terms_and_conditions_accepted = datetime.now()
 
```

Comment:
I would add something like a "change password email" feature at creation time, what do you think?